provider/pkg/gen/go-templates/helm/v2/chart.tmpl

```diff
@@ -339,7 +339,7 @@ func parseChart(ctx *pulumi.Context, name string, args chartArgs, opts ...pulumi
 		return nil, err
 	}
 
-	if v3 {
+	if v3 && args.InstallCRDs != false {
```

Comment:
```suggestion
	if v3 && args.InstallCRDs {
```

Reply:
@lblackstone I set it this way because I believe we want this to be `true` by default, and I don't see a way to set it, any ideas?